db/migrate/20201219201758_change_availability_date_data_type.rb

```diff
@@ -1,10 +0,0 @@
-class ChangeAvailabilityDateDataType < ActiveRecord::Migration[6.1]
-  def change
-    change_table :availabilities do |t|
-      t.remove :start_date_time, :end_date_time, :users_id
-      t.references :user, null: false, foreign_key: true
-      t.string :start_date_time
-      t.string :end_date_time
-    end
-  end
-end
```

Comment:
Ended up changing some existing migrations so you may wanna reset your db (drop, create, migrate) 🙏🏻